src/main/groovy/com/ullink/AssemblyInfoVersionPatcher.groovy

```diff
@@ -64,7 +64,10 @@ class AssemblyInfoVersionPatcher extends ConventionTask {
             replace(it, 'AssemblyVersion', getVersion())
             replace(it, 'AssemblyFileVersion', getFileVersion())
             replace(it, 'AssemblyInformationalVersion', getInformationalVersion())
-            replace(it, 'AssemblyDescription', getAssemblyDescription())
```

Comment:
maybe we could do that for each replacement, not just the description ? (i.e. if empty skip it)
WDYT ?